cmd/restic/cmd_mount.go

```diff
@@ -141,9 +142,16 @@ func runMount(ctx context.Context, opts MountOptions, gopts GlobalOptions, args
 		return err
 	}
 
+	repositoryID, err := repo.ID()
+	if err != nil {
+		return err
+	}
+
+	fuseMountName := fmt.Sprintf("restic:%s", repositoryID.Str())
```

Comment:
I'd prefer to use `s.Config().ID[:10]` to get the short repository ID. That snippet is already used to fill in the ID in the `"created restic repository %v at %s"` message printed on startup.

Reply:
I have updated the mount point to use `repo.Config().ID[:10]` instead of a new method which parses the repository ID.